code/game/machinery/doors/airlock_control.dm

```diff
@@ -129,20 +129,14 @@
 		frequency = new_frequency
 		radio_connection = SSradio.add_object(src, frequency, RADIO_AIRLOCK)
 
+// this is an override, in addition to the one in airlock_control.dm
```

Comment:
I decided not to merge it because it makes sense to keep signals-related stuff here and everything else there.
The `..()` does call the other override just fine.